server/views/pages/licence/partials/title.njk

```diff
@@ -0,0 +1,5 @@
+{% macro title(offenderName) %}
+    <div class="title" id="title">
+        Licence for {{ offenderName }}<br/><br/>
```

Comment:
`<br>` (i.e. styling as markup) is generally frowned upon.. adding `margin-bottom` to the title class is more accessible and browser friendly

Reply:
I agree. Thought I'd removed any that were left. Will sort.

Reply:
Removed & styled.